docs/custom-operator-image.md

```diff
@@ -0,0 +1,22 @@
+# Customer Operator container image
+
+In some case, building a customer Operator image is needed, for example: to add a customer secret backend implementation, or to add CA certificat.
```

Comment:
Can you provide some more context about what a custom operator image is and explain how it's used in Datadog? This sentence jumps right into "In some cases..." but it's not clear what a user might be doing at the time they'll need to build a customer operator image.